invoicing/pdf.py

```diff
@@ -46,20 +62,21 @@ def get_cached_apartment(apartment_uuid: UUID) -> ApartmentDocument:
     invoice_pdf_data_list = []
     for installment in installments:
         reservation = installment.apartment_reservation
-        profile = reservation.application_apartment.application.customer.primary_profile
+        payer_name_and_address = _get_payer_name_and_address(
+            installment.apartment_reservation.customer
+        )
         apartment = get_cached_apartment(reservation.apartment_uuid)
         project = get_cached_project(apartment.project_uuid)
         invoice_pdf_data = InvoicePDFData(
             recipient=project.project_housing_company,
-            recipient_account_number=installment.account_number,
-            payer_name_and_address=f"{profile.first_name} {profile.last_name}\n\n"
-            f"{profile.street_address}\n"
-            f"{profile.postal_code} {profile.city}",
+            recipient_account_number=f"{apartment.project_contract_rs_bank} "
```

Comment:
This won't work unless `get_apartment(apartment_uuid)` above is changed to `get_apartment(apartment_uuid, include_project_fields=True)`.

Also, not sure if this will be an issue still in production, but at least atm it is possible that the field isn't filled, and we get `None` printed on the invoice, which is not very nice 🙂 Not sure though if a missing bank name should be just ignored or should it be an error 🤔

Reply:
Fixed that, actually I can just use `project.project_contract_rs_bank`
Well, I don't know if the data is missing in ES or if it's just that we have never parsed it to `ApartmentDocument` before. The only source I can find info about this field is from you :D 
But I think it should be ignored instead of throwing an error